includes/admin/load.php

```diff
@@ -38,7 +38,7 @@ function perflab_add_features_page() {
  * Initializes functionality for the features page.
  *
  * @since 1.0.0
- * @since n.e.x.t Renamed to perflab_load_features_page(), and the
+ * @since 3.0.0 Renamed to perflab_load_features_page(), and the
  *                $module and $hook_suffix parameters were removed.
```

Comment:
```suggestion
 * @since 3.0.0 Renamed to perflab_load_features_page(), and the
 *              $module and $hook_suffix parameters were removed.
```